ivy/functional/frontends/torch/tensor.py

```diff
@@ -367,6 +367,11 @@ def log_(self):
     def log2(self):
         return torch_frontend.log2(self)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
+    def log2_(self):
+        self.ivy_array = self.log2().ivy_array
+        return self
```

Comment:
Not sure I understand what you are trying to do here. Can't you just return `torch_frontend.log2(self)`?

Reply:
In the `log2(self)` function just above it we are returning `torch_frontend.log2(self)`. In this function, We are calling that above function and updating the array itself.